surfer/io.py

```diff
@@ -225,7 +225,7 @@ def project_volume_data(filepath, hemi, reg_file=None, subject_id=None,
     out = p.returncode
     if out:
         raise RuntimeError(("mri_vol2surf command failed "
-                            "with command-line: ") + " ".join(cmd_list))
```

Comment:
The command gets printed here either way: https://github.com/nipy/PySurfer/pull/172/files#diff-bd3cabf83a69fe1b0d92a1df82fc1241R222